fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/EverythingOperationTest.java

```diff
@@ -180,7 +180,7 @@ public void testPatientEverythingWithCount() {
         Bundle everythingBundle = response.readEntity(Bundle.class);
```

Comment:
on 177, I'd expect some header settings, like the following.

```
Response response =
                target.path("Organization").request()
                .header("X-FHIR-TENANT-ID", tenantName)
                .header("X-FHIR-DSID", dataStoreId)
                .post(entity, Response.class);
        assertResponse(response, Response.Status.CREATED.getStatusCode());
```

Reply:
Got it.  Fixed.